CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [v15.0.0] 2023-03-10
+
```

Comment:
Should this include migration instructions from **CUMULUS-3147** re: cumulus-ecs-task images and/or async-operation images configuration values?

Reply:
I didn't create the document since there are only two minor changes in the PR #3228.  We should consider writing instructions for upgrading

Reply:
@jennyhliu  That's fine, but users should probably be aware they need to upgrade the image configuration in their deployments if they specify a default right?